_posts/_posts/0000-01-02-Daxtyn.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
Click the Files Changed tab in this pull request
Edit your newly added file
Replace line 5 with something new
Scroll to the bottom and click Commit Changes